packages/ui/src/app/pages/WorkingGroups/WorkingGroup/WorkingGroup.stories.tsx

```diff
@@ -20,12 +25,46 @@ const WG_DATA = {
   name: 'membership',
 }
 
+const WG_OPENING_DATA = {
```

Comment:
To make things clearer please:

1. Rename this to:
```suggestion
const WG_OPENING_METADATA = {
```